src/runtime/analytics-service.js

```diff
@@ -80,6 +66,12 @@ export class AnalyticsService {
      * @private @const {!TransactionId}
      */
     this.xid_ = new TransactionId(deps);
+
+    /** @private {?Promise} */
```

Comment:
Now it should be `{?Promise<!ActivityPort>}`

Reply:
done